tests/run/i15318.check

```diff
@@ -0,0 +1,8 @@
+inspecting field value
+interface JsonProperty
```

Comment:
Is that expected? Shouldn't `@beenGetter` limit it to the bean getter and remove it from the field?

Reply:
I don't believe so.  The meta package documentation says "The meta-annotations in package `scala.annotation.meta` are used to control where annotations on fields and class parameters are copied.", so I removing from the field isn't expected.

Reply:
Is that by defacto of Scala 2 implementation?  Because otherwise that seems contrary to the documentation.  Documentation bug?

Reply:
Yeah, it's what Scala 2 does, the documentation should be more clear. The goal is to be able to control where the annotation goes, which would not be possible if it always goes to the field.